README.md

```diff
@@ -165,5 +175,7 @@ relationships between the two are set up in Active Record.
 
 Write your code here:
 ```ruby
-# code here
+genie = Genie.create(name: "Genie")
+lamp = Lamp.create(wishes_remianing: 3)
+lamp.update(genie: genie)
 ```
```

Comment:
There a few more steps left. Do you think you would be able to write them out after more time with Ruby and ActiveRecord?